src/css/components/_main-menu.scss

```diff
@@ -82,8 +82,7 @@
   & .menu-item {
     padding: 0 16px;
     width: 100%;
-    // I want this to be min-height, but IE breaks layout if I do
-    height: 48px;
+    min-height: 48px;
```

Comment:
@jakearchibald I went with what you wanted in the comment. That being said, not 100% sure this doesn't cause a small reflow. Assuming the menu items can never be taller than 48px, I'd personally stick with `height` :)

Reply:
`min-height` seems better to me, but it doesn't massively matter with the current UI.

Before:

![jakearchibald github io_svgomg_ (2)](https://user-images.githubusercontent.com/93594/133261520-8ad66686-bd30-42eb-b0ec-b8e137af4fe2.png)

After:

![deploy-preview-271--affectionate-lovelace-6ea273 netlify app_](https://user-images.githubusercontent.com/93594/133261536-7d06c3b5-3af6-45d7-b569-b09a4950dc68.png)

Reply:
Yeah, I know that, but with the current UI it's not needed is my point. Anyway, your call; if you don't see any reflows or anything bad, we can go with `min-height`, otherwise I can switch back to `height`

Reply:
`min-height` shouldn't cause a visual reflow here, since it'll know the height of each item in a single layout.

Reply:
Though, I do personally see a reflow when Ctrl+F5. Maybe reflow isn't the right term, but the `.menu-item` is resized. Using the latest Edge/Chrome.